internal/db/collection_acp.go

```diff
@@ -36,10 +37,9 @@ func (c *collection) registerDocWithACP(
 	if !c.db.acp.HasValue() {
 		return nil
 	}
-	identity := GetContextIdentity(ctx)
 	return permission.RegisterDocOnCollectionWithACP(
 		ctx,
-		identity,
+		identity.FromContext(ctx),
```

Comment:
question: Why was this needed?

Reply:
nothing changed here, akhi. `identity` used to be a variable, now it's package.